resolwe_bio/processes/workflows/cutadapt_star_fc_quant_tf.yml

```diff
@@ -0,0 +1,171 @@
+- slug: workflow-cutadapt-star-fc-quant-single
+  name: Cutadapt - STAR - FeatureCounts (3' mRNA-Seq, single-end)
+  data_name: "{{ reads|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:quant:featurecounts:single
+  category: Pipeline
+  description: |
+    This 3' mRNA-Seq pipeline is comprised of QC, preprocessing,
+    alignment and quantification steps.
+
+    Reads are preprocessed by __Cutadapt__ which removes adapters, trims
+    reads for quality from the 3'-end, and discards reads that are too short
+    after trimming. Preprocessed reads are aligned by __STAR__
+    aligner. For read-count quantification, the __FeatureCounts__ tool
+    is used.
+
+    QC steps include downsampling, QoRTs QC analysis and alignment of
```

Comment:
Consider rephrasing the downsampling part. Downsampling does not contribute any information, i.e. report, to QC. I would therefore make a distinction from the rest of the tools. Consider using something along the lines _QC operates on downsampled reads and includes ..._